src/main/java/databaseInteracting/Client.java

```diff
@@ -181,11 +177,25 @@ public static void main(String[] args) throws SQLException, IOException {
         int importOption = 0;
         do {
           System.out.println("Select the type of file you desire:");
-          System.out.println("[1] - Parquet file. ");
-          System.out.println("[2] - Json file. ");
+          System.out.println("[1] - Json file. ");
+          System.out.println("[2] - Parquet file. ");
           importOption = scanner.nextInt();
           if (importOption == 1) {
-            System.out.println("Invoke method to import database do Parquet file");
+            System.out.println("Reading data from the JSON file and store on the database.");
+            byte[] mapData = Files.readAllBytes(Paths.get(jsonExportFileName));
+            Product[] productsArray = null;
+            ObjectMapper objectMapper = new ObjectMapper();
+            productsArray = objectMapper.readValue(mapData, Product[].class);
+            Product[] productList = productsArray;
+            for (Product product : productList) {
+              String name = product.getName();
+              int stock = product.getStock();
+              float price = product.getPrice();
+              AddProductRequest request =
+                  AddProductRequest.newBuilder().setName(name).setStock(stock).setPrice(price).build();
+              AddProductResponse response = prodStub.addProduct(request);
+              System.out.println(response.getName());
+            }
```

Comment:
Try transforming this into a ```IntStream```

Reply:
This way you can practice a new tool for you

Reply:
This is not what I was expecting, but still a valid solution for a small project!
Be aware of the implications of this approach:

By reading the JSON file on client side, we make this a client-only feature, so all the importing logic is responsibility of the client and the server doesn't even know that "importing" is a thing.

1. If you need to implement a new file format, you make your package bigger (because it has more dependencies).
2. If you have multiple clients, all of them will need to implement this same feature, for every file format you want to support.
3. It is slow: the client need to call `addProduct` one by one, if implemented on server, you could add them in a single SQL statement